### DISCUSSION TITLE
Type error in defining Date scalars with DateResolver

### DISCUSSION THREAD

**Discussion by Yedidya10** [hayes/pothos#828] [Q&A] [answered]
Hi,

I get the following error and I don't know what the problem is.

I basically copied the code from this [project](https://youtu.be/g6JPWcU9tjQ?t=411).
![Type error in defining Date scalars with DateResolver](https://user-images.githubusercontent.com/86730512/223529533-2281cb58-24a4-443f-8377-49d37ffd7352.png)

**Comment by hayes** [comment, marked as answer]
It looks like scalars in your builder generic should be capitalized.  Let me know if that fixes your issue.

**Reply by Yedidya10** [reply]
Can you give me an example of what you mean? Because I tried all kinds of variations and it didn't solve the problem.

**Reply by hayes** [reply]
https://stackblitz.com/edit/typescript-bul2zn?file=index.ts
